dist/edu-benefits-schema.json

```diff
@@ -504,25 +499,56 @@
       "$ref": "#/definitions/dateRange"
     },
     "serviceBefore1977": {
-      "type": "boolean"
+      "type": "object",
+      "properties": {
+        "married": {
+          "type": "boolean"
+        },
+        "haveDependents": {
+          "type": "boolean"
+        },
+        "parentDependent": {
+          "type": "boolean"
+        }
+      },
+      "required": [
+        "married",
+        "haveDependents",
+        "parentDependent"
+      ]
     },
     "remarks": {
       "type": "string"
     },
     "toursOfDuty": {
       "type": "array",
       "items": {
-        "dateRange": {
-          "$ref": "#/definitions/dateRange"
-        },
-        "serviceBranch": {
-          "type": "string"
-        },
-        "serviceStatus": {
-          "type": "string"
-        },
-        "involuntarilyCalledToDuty": {
-          "type": "boolean"
+        "type": "object",
+        "properties": {
+          "dateRange": {
+            "$ref": "#/definitions/dateRange"
+          },
+          "serviceBranch": {
+            "type": "string"
+          },
+          "serviceStatus": {
+            "type": "string"
+          },
+          "benefitsToApplyTo": {
+            "type": "array",
+            "items": {
+              "type": "string",
+              "enum": [
+                "chapter33",
+                "chapter30",
+                "chapter1606",
+                "chapter32"
+              ]
+            }
+          },
+          "involuntarilyCalledToDuty": {
+            "type": "boolean"
```

Comment:
On the front end, this has three options: Yes, No, and Not Applicable. Should see if we need that third option, probably.